src/features/recruitments/components/filter-panel.tsx

```
@@ -0,0 +1,124 @@
+  return (
+    <div className="flex gap-2">
+      <button
+        onClick={onReset}
+        disabled={disableReset}
+        className={cn(
+          'flex-1 rounded-lg border py-2.5 text-sm font-medium transition-colors',
+          disableReset
+            ? 'border-border text-muted-foreground/40 cursor-default'
+            : 'text-muted-foreground hover:text-foreground cursor-pointer',
+        )}
+      >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)